scripts/OGC_tests/launch_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
For some reason this is not working for me if I want to launch tests locally without running the installation script again. But I guess it's fine for now.